beacon-chain/db/slasherkv/schema.go

```diff
@@ -9,7 +9,6 @@ package slasherkv
 var (
 	// Slasher buckets.
 	attestedEpochsByValidator  = []byte("attested-epochs-by-validator")
-	attestationRecordsBucket   = []byte("attestation-records")
```

Comment:
We should add a database "migration" to delete this bucket to reclaim the space, if it is no longer used. 

@rauljordan, what do you think?

Reply:
This db is unused at the moment, so we are safe, but we should add a migration in the future for slasher changes

Reply:
I had the same question. But @rauljordan clarified that we do migration in another PR as not many people use it